spacify-portal/src/main/java/org/uci/spacifyPortal/controllers/RuleController.java

```diff
@@ -0,0 +1,25 @@
+package org.uci.spacifyPortal.controllers;
+
```

Comment:
this controller should be merged with the RoomController. All the controller APIs that operate on our room object should be put under the same Controller. 
We should have the RoomContrroller with RequestMapping as api/v1/room
and this should be a sub controller as /addRule.